gallery/how_to/use_tvms_python_api/tvmc_python.py

```diff
@@ -0,0 +1,236 @@
+# ------------------------------------------------
+# 
+#  .. code-block:: python
+#      python my_tvmc_script.py
+# 
+# Note: Your fans may become very active
+# 
+
+################################################################################
+# Example results:
+# ----------------
+# 
+#   .. code-block:: python
+#      Time elapsed for training: 18.99 s
+#      Execution time summary:
+#      mean (ms)   max (ms)   min (ms)   std (ms) 
+#        25.24      26.12      24.89       0.38 
+#      
+#      Output Names:
+#       ['output_0']
+#
+
+Additional TVMC Functionalities
+===============================
+
+################################################################################
+# Saving the model
+# ----------------
+# 
+# To make things faster for later, after loading the model (Step 1) save the relay version. 
+# 
+#   .. code-block:: python
+#      model = tvmc.load('my_model.onnx') #Step 1: Load
+#      model.save(model_path) 
+# 
+
+################################################################################
+# Saving the package
+# ------------------
+# 
+# After the model has been compiled (Step 2) the package also is also saveable.  
+# 
+#    .. code-block:: python
+#      tvmc.compile(model, target="llvm", package_path="whatever") #Step 2: Compile
+#      
+#      new_package = tvmc.TVMCPackage(package_path="whatever") 
+#      result = tvmc.run(new_package) #Step 3: Run
+# 
+
+################################################################################
+# Using Autoscheduler
+# -------------------
+# Use the next generation of tvm to enable potentially faster run speed results. 
+# The search space of the schedules is automatically generated unlike 
+# previously where they needed to be hand written. (Learn more: 1, 2) 
+# 
+#    .. code-block:: python
+#      tvmc.tune(model, target="llvm", enable_autoscheduler = True) #Step 1.5: Optional Tune
+#
+
+################################################################################
+# Saving the tuning results
+# -------------------------
+# 
+# The tuning results can be saved in a file for later reuse.
+# 
+# Method 1:
+#    .. code-block:: python
+#      log_file = "hello.json"
+#      
+#      # Run tuning
+#      tvmc.tune(model, target="llvm",tuning_records=log_file)
+#      
+#      ...
+#      
+#      # Later run tuning and reuse tuning results
+#      tvmc.tune(model, target="llvm",tuning_records=log_file)
+#  
+# Method 2:
+#    .. code-block:: python
+#      # Run tuning
+#      tuning_records = tvmc.tune(model, target="llvm")
+#      
+#      ...
+#      
+#      # Later run tuning and reuse tuning results
+#      tvmc.tune(model, target="llvm",tuning_records=tuning_records)
+# 
+
+################################################################################
+# Tuning a more complex model:
+# ----------------------------
+# If you notice T's (timeouts) printed, increase the searching time frame: 
```

Comment:
Please add a bit more context here, perhaps an example of a timeout in the command line where you can point to the timeouts being mentioned here.